EIPS/eip-884.md

```diff
@@ -254,40 +254,40 @@ This will be extended as follows:
         function isSuperseded(address addr) public view returns (bool);
 
         /**
-         *  Gets the most recent address given a superseded one.
+         *  Gets the most recent address, given a superseded one.
          *  Addresses may be superseded multiple times, so this function needs to
          *  follow the chain of addresses until it reaches the final, verified address.
          *  @param addr The superseded address.
-         *  @return the verified address that ultimately holds the stock.
+         *  @return the verified address that ultimately holds the share.
          */
         function getCurrentFor(address addr) public view returns (address);
     }
 
-### SEC Requirements
+### Securities Exchange Commission Requirements
```

Comment:
If SEC is to be used in the following paragraph, should this not be
`### Securities Exchange Commission (SEC) Requirements`?

Reply:
yes. fixed.  thanks